docs/variables.md

```diff
@@ -85,22 +85,22 @@ For instructions on using variable substitutions see the relevant section of [th
 | `Task`        | `spec.steps[].script`                                           |
 | `Task`        | `spec.steps[].onError`                                          |
 | `Task`        | `spec.steps[].env.value`                                        |
-| `Task`        | `spec.steps[].env.valuefrom.secretkeyref.name`                  |
-| `Task`        | `spec.steps[].env.valuefrom.secretkeyref.key`                   |
-| `Task`        | `spec.steps[].env.valuefrom.configmapkeyref.name`               |
-| `Task`        | `spec.steps[].env.valuefrom.configmapkeyref.key`                |
-| `Task`        | `spec.steps[].volumemounts.name`                                |
-| `Task`        | `spec.steps[].volumemounts.mountpath`                           |
-| `Task`        | `spec.steps[].volumemounts.subpath`                             |
+| `Task`        | `spec.steps[].env.valueFrom.secretKeyRef.name`                  |
```

Comment:
It should actually be camel case notation.

Reply:
This will go away once this is rebased on top of the related PR, right?

Reply:
Yes, that's why I set up the `do-not-merge/hold`.